src/main/java/tutorlink/lists/ComponentList.java

```diff
@@ -25,7 +25,7 @@ public ComponentList findComponent(String name) throws ComponentNotFoundExceptio
         ComponentList filteredList = new ComponentList();
         filteredList.componentArrayList = componentArrayList
                 .stream()
-                .filter(comp -> comp.getName().contains(name))
+                .filter(comp -> comp.getName().toUpperCase().contains(name.toUpperCase()))
```

Comment:
Just to check, the component names are converted to uppercase so that the search is not case-sensitive?

Reply:
Yes that is correct